ocs_ci/utility/utils.py

```diff
@@ -2280,3 +2285,41 @@ def add_stage_cert():
         f"oc patch image.config.openshift.io cluster --type=merge"
         f" -p '{additional_trusted_ca_patch}'"
     )
+
+
+def get_module_ip(terraform_state_file, module):
+    """
+    Gets the bootstrap node IP from terraform.tfstate file
+
+    Args:
+        terraform_state_file (str): Path to terraform state file
+        module (str): Module name in terraform.tfstate file
+            e.g: constants.LOAD_BALANCER_MODULE
+
+    Returns:
+        str: IP of bootstrap node
+
+    """
+    with open(terraform_state_file) as fd:
+        obj = hcl.load(fd)
+
+        resources = obj['resources']
+        log.debug(f"Extracting module information for {module}")
+        log.debug(f"Resource in {terraform_state_file}: {resources}")
+        for resource in resources:
+            if resource['module'] == module:
+                resource_body = resource['instances'][0]['attributes']['body']
+                return resource_body.split("\"")[3]
+
+
+def set_aws_region(region=None):
```

Comment:
Where this function is used? In case you need to execute some command using subprocess you can add whatever env variables to proper parameter, so maybe this is not needed? BTW I don't see it's used anywhere in your code or did I missed it somewhere?